pubsub/lib/mix/tasks/pubsub.ex

```diff
@@ -0,0 +1,70 @@
+defmodule Mix.Tasks.Pubsub do
+  use Mix.Task
+
+  @shortdoc "Runs a simple publish/subscribe example."
+  def run(_) do
+    {:ok, _started} = Application.ensure_all_started(:goth)
+    :httpc.set_options(pipeline_timeout: 1000)
+    project_id =
+      "Project ID?: "
+      |> IO.gets()
```

Comment:
This IO.gets() |> String.trim("\n") pattern is happening many times. Maybe create a small helper function for it?

```elixir
defp get_response(prompt) do
  prompt |> IO.gets() |> String.trim("\n")
end

...
project_id = get_response("Project ID?: ")
```

Reply:
But the pipe chain looks so cool! OK, will do.